### PR TITLE
Make copy resources echoes always return true

### LINKED ISSUE #6595: Pods-<Target>-resources.sh echo commands generate output too large for Xcode
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

While this really is a Xcode bug, it is unlikely to be fixed any time soon, if ever, and thus CocoaPods needs to work around it.

## What did you do?

Build workspace

## What did you expect to happen?

Build succeeds

## What happened instead?

Pods-Target-resources.sh script fails with error due to `echo` on line 48:
```
error: Resource temporarily unavailable
Command /bin/sh failed with exit code 1
```
### Stack

```
   CocoaPods : 1.2.0
        Ruby : ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]
    RubyGems : 2.5.2
        Host : Mac OS X 10.12.3 (16D32)
       Xcode : 8.2.1 (8C1002)
         Git : git version 2.10.2
Ruby lib dir : /usr/local/Cellar/ruby/2.3.3/lib
Repositories : cocoapods - https://github.com/cocoapods/cocoapods.git @ 6a6df6f01d48b769bb89bc9f7d26bbc312624740
               LPM - git@github.com:lonelyplanet/lpm-specs-ios.git @ 2a89500ce94755688ab621a4dc9a8ce5081dd063
               master - https://github.com/CocoaPods/Specs.git @ 782b74bf2f9c3c131e41a105785e914d046dfdbd
```

### Installation Source

```
Executable Path: /Users/alondon/.gems/bin/pod
```

### Plugins

```
cocoapods-deintegrate   : 1.0.1
cocoapods-plugins       : 1.0.0
cocoapods-search        : 1.0.0
cocoapods-sorted-search : 0.2.4
cocoapods-stats         : 1.0.0
cocoapods-trunk         : 1.1.2
cocoapods-try           : 1.1.0
```

## Project that demonstrates the issue

In any project using CocoaPods replace all `echo` commands that go to stdout in `Pods/Target Support Files/Pods-Target/Pods-Target-resources.sh` with:

```
echo `printf '=%.0s' {1..600}` 
```

Appending `|| true` to those lines works around the issue.

### PR DESCRIPTION
**Pull request by postmechanical**
Resolves #6595

### CODE REVIEW

**Comment by postmechanical** [comment]
@dnkoutso 👀 🙏

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! 
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Make copy resources echoes always return true 
  [postmechanical](https://github.com/postmechanical)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Alfredo Delli Bovi](https://github.com/adellibovi)
   [#6033](https://github.com/CocoaPods/CocoaPods/issues/6033)
 
+* Make copy resources echoes always return true  
+  [postmechanical](https://github.com/postmechanical)
+  [#6595](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```
Comment (DanToml) [inline CHANGELOG.md]: could you update the URL to include the issue number?

Comment (postmechanical) [inline CHANGELOG.md]: @DanToml Oops...done! Added the radar number to the comment as well.

**Comment by dnkoutso** [comment]
@postmechanical to get this green we will need to update https://github.com/CocoaPods/cocoapods-integration-specs

Do you have everything cloned? Run `bundle exec rake spec:rebuild_integration_fixtures` which should update the files and then make a PR on that repo. Then we will update this PR to bump the submodule reference.

**Comment by postmechanical** [comment]
@dnkoutso Rebased upstream

**Comment by dnkoutso** [comment]
@postmechanical are you sure this is the right rebase? The diff doesnt show your changes.

**Comment by postmechanical** [comment]
@dnkoutso Definitely doesn't look right. This is what I did:

```
git fetch upstream
remote: Counting objects: 31, done.
remote: Compressing objects: 100% (31/31), done.
remote: Total 31 (delta 9), reused 1 (delta 0), pack-reused 0
Unpacking objects: 100% (31/31), done.
From github.com:CocoaPods/CocoaPods
 * [new branch]      0.37-stable                        -> upstream/0.37-stable
 * [new branch]      0.38-stable                        -> upstream/0.38-stable
 * [new branch]      0.39-stable                        -> upstream/0.39-stable
 * [new branch]      1-0-stable                         -> upstream/1-0-stable
 * [new branch]      1-1-stable                         -> upstream/1-1-stable
 * [new branch]      1-2-stable                         -> upstream/1-2-stable
 * [new branch]      ali-xcassets-1546                  -> upstream/ali-xcassets-1546
 * [new branch]      dan_xcode_8_lint                   -> upstream/dan_xcode_8_lint
 * [new branch]      dani_bundle_update_xcodeproj       -> upstream/dani_bundle_update_xcodeproj
 * [new branch]      dani_circleci_examples             -> upstream/dani_circleci_examples
 * [new branch]      dani_swift_pod_variance            -> upstream/dani_swift_pod_variance
 * [new branch]      dani_xcode_8_examples_take_3       -> upstream/dani_xcode_8_examples_take_3
 * [new branch]      delay-executable-result-with-pty   -> upstream/delay-executable-result-with-pty
 * [new branch]      fabio-pods-from-projects           -> upstream/fabio-pods-from-projects
 * [new branch]      k0nserv-git-fail-gracefully        -> upstream/k0nserv-git-fail-gracefully
 * [new branch]      kylef/circleci                     -> upstream/kylef/circleci
 * [new branch]      local_repos                        -> upstream/local_repos
 * [new branch]      master                             -> upstream/master
 * [new branch]      mr-fix-private-header-search-paths -> upstream/mr-fix-private-header-search-paths
 * [new branch]      mr-issuetemplate                   -> upstream/mr-issuetemplate
 * [new branch]      mr-speedup-embed-frameworks        -> upstream/mr-speedup-embed-frameworks
 * [new branch]      orta-create-ix                     -> upstream/orta-create-ix
 * [new branch]      pods-project-edit-feature          -> upstream/pods-project-edit-feature
 * [new branch]      seg-auto-update-specs-repo         -> upstream/seg-auto-update-specs-repo
 * [new branch]      seg-concurrent                     -> upstream/seg-concurrent
 * [new branch]      seg-fix-podspec                    -> upstream/seg-fix-podspec
 * [new branch]      seg-infer-umbrella-header          -> upstream/seg-infer-umbrella-header
 * [new branch]      seg-no-embed-in-extensions         -> upstream/seg-no-embed-in-extensions
 * [new branch]      seg-regressions                    -> upstream/seg-regressions
 * [new branch]      seg-rspec                          -> upstream/seg-rspec
 * [new branch]      seg-sandbox-profile-escape         -> upstream/seg-sandbox-profile-escape
 * [new branch]      seg-skip-import-validation         -> upstream/seg-skip-import-validation
 * [new branch]      seg-xcode-7                        -> upstream/seg-xcode-7
 * [new branch]      stable-0.36                        -> upstream/stable-0.36
 * [new branch]      stable-0.36.1                      -> upstream/stable-0.36.1
 * [new tag]         1.2.1.rc.1                         -> 1.2.1.rc.1
git rebase upstream/master
First, rewinding head to replay your work on top of it...
Applying: Make copy resources echoes always return true
Applying: Add changelog entry
Applying: Add issue number to URL and radar number to comment
``

**Comment by dnkoutso** [comment]
@postmechanical your submodule reference should point to `11064dbaee00b46980afd8589a428d3d1a06108b`

**Pull request closed without merging**